filebeat/docs/modules/kibana.asciidoc

```diff
@@ -23,16 +23,18 @@ include::../include/running-modules.asciidoc[]
 
 include::../include/configuring-intro.asciidoc[]
 
-//set the fileset name used in the included example
+//set the fileset name used in the included file
 :fileset_ex: log
 
 include::../include/config-option-intro.asciidoc[]
 
 [float]
-==== `{fileset}` log fileset settings
+==== `log` fileset settings
 
 include::../include/var-paths.asciidoc[]
 
+:fileset_ex!:
```

Comment:
@dedemorton What does this actually do? It doesn't _seem_ to be having any visible effect in the generated HTML that I'm seeing:

![screen shot 2018-08-14 at 4 11 25 pm](https://user-images.githubusercontent.com/51061/44123355-b9fc3440-9fdc-11e8-8eb4-88d3c4542947.png)

Reply:
@ycombinator It clears the attribute value so that it doesn't inadvertently get carried into the next topic. The attribute value will be in scope within the book until it's either cleared or reset, so clearing the value is a best practice unless you want to value to exist in other topics (and in this case, you do not).

Reply:
Gotcha, thanks for explaining that. From what I see after that attribute is set on line 27, it is used by the included file on line 29 (`config-option-intro.asciidoc`). It is not, however, used by the included file on line 34 (`var-paths.asciidoc`). So does it make sense to clear the attribute right after line 29?

Reply:
I don't think it really matters. Clearing everything at the end is probably easier for people updating the files (they are less likely to accidentally delete the entries...maybe).

Reply:
Yeah, fair point. I see you're doing that with `modulename` too. I'm good with keeping things as-is.